src/ui/public/index_patterns/_field_format/contentTypes.js

```diff
@@ -6,6 +6,10 @@ export default function contentTypesProvider(highlightFilter) {
   var types = {
     html: function (format, convert) {
       return function recurse(value, field, hit) {
+        if (typeof value === 'undefined') {
```

Comment:
if this is converted to `value == null` then it might be a little clearer (I missed the typeof value at first, because we rarely do this type of check) and it will catch `null` along with `undefined`, which seems like a win to me.